x/bank/keeper/send.go

```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/pkg/errors"
```

Comment:
Why isn't it simply the stdb errors package?

Reply:
Thanks for noticing, must have been my IDE pulling the wrong import. Fixed.